public/theme/extensions-2021/partials/_header.twig

```diff
@@ -0,0 +1,46 @@
+<header class="site-header">
+    <div class="wrapper site-header__wrapper">
+        <a class="navbar-brand" href="/">
+            <img class="logo" width="200" height="43" src="{{ asset('img/logo.svg') }}" alt="Logo Bolt">
+        </a>
+        <div class="site-header__middle">
+            <nav class="nav">
+                <button class="nav__toggle" aria-expanded="false" type="button">
+                    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd"
+                              d="M4 7C4 6.44772 4.44772 6 5 6H19C19.5523 6 20 6.44772 20 7C20 7.55228 19.5523 8 19 8H5C4.44772 8 4 7.55228 4 7ZM4 12C4 11.4477 4.44772 11 5 11H19C19.5523 11 20 11.4477 20 12C20 12.5523 19.5523 13 19 13H5C4.44772 13 4 12.5523 4 12ZM4 17C4 16.4477 4.44772 16 5 16H19C19.5523 16 20 16.4477 20 17C20 17.5523 19.5523 18 19 18H5C4.44772 18 4 17.5523 4 17Z"
+                              fill="#282828"/>
+                    </svg>
+                </button>
+                <ul class="nav__wrapper">
+                    {% for item in menu_array() %}
+                        <li class="nav__item {{ item.class }} {% if item.submenu|default %}has-submenu{% endif %}">
+                            <a {% if item == menu_array()|last %} transition-style="in:circle:top-right" {% endif %}
+                                    href="{{ item.link }}" {% if item.submenu|default %} aria-haspopup="true" {% endif %}
+                                    {% if item.target is defined %}target="{{ item.target|escape }}"{% endif %}>{{ item.label }}</a>
+                            {% if item.submenu|default %}
+                                <div>
+                                    <ul class="dropdown" aria-label="submenu">
+                                        {% for subitem in item.submenu %}
+                                            <li>
+                                                <a href="{{ subitem.link }}">{{ subitem.label }}</a>
+                                            </li>
+                                        {% endfor %}
+                                    </ul>
+                                </div>
+                            {% endif %}
+                        </li>
+                    {% endfor %}
+                    <div class="search-container">
```

Comment:
As discussed, let's make sure that the search field here and on the search page uses the styles that Derk made for https://boltcms.io/submit-showcase

Reply:
😎